spk/transmission/src/service-setup.sh

```diff
@@ -1,16 +1,13 @@
+# use system python for DSM7
+PYTHON_BIN_PATHS=""
+
```

Comment:
I would ommit this variable since it is a leftover of DSM 6 compatibility.

OR - it could be `PYTHON_BIN_PATH=$(which python3)` but this would be obsolete too...

OR - it could be used as precondition. If $(which python3) is empty, then the package should abort the installation.

OR - if python3 is not found in the path, take a python path from the user (in a wizard).


Another approach would be to write the python3 path into a config file at time of installation, and load this file in service-setup.sh to add the value of the variable to the PATH below. This enables an option for the user to manually configure (overwrite) the python path.
This is useful, when a python script depends on a newer python version than what Synology provides with DSM (i.e. you could configure the path of our python310 or python311 package).

Reply:
I leave it to your discretion, no opinion here.

Reply:
@mreid-tt I removed the python path at all.

To execute python scripts with a non default python3, one can adjust the shebang in the script.